repo/fsrepo_test.go

```diff
@@ -82,20 +83,26 @@ const (
 }`
 )
 
-func TestInitRepo(t *testing.T) {
+func TestInitRepoDirect(t *testing.T) {
 	tf.UnitTest(t)
 	cfg := config.NewDefaultConfig()
 
+	// Inits a repo and opens it (ensuring it is openable)
+	initAndOpenRepoDirect := func(repoPath string, version uint, cfg *config.Config) (*FSRepo, error) {
+		if err := InitFSRepoDirect(repoPath, version, cfg); err != nil {
+			return nil, err
+		}
+		return OpenFSRepo(repoPath, version)
+	}
+
 	t.Run("successfully creates when directory exists", func(t *testing.T) {
 		dir, err := ioutil.TempDir("", "init")
 		assert.NoError(t, err)
-		defer func() {
-			require.NoError(t, os.RemoveAll(dir))
-		}()
+		defer RequireRemoveAll(t, dir)
 
-		_, err = initAndOpenRepo(dir, cfg)
+		_, err = initAndOpenRepoDirect(dir, 42, cfg)
 		assert.NoError(t, err)
-		checkNewRepoFiles(t, dir)
+		checkNewRepoFiles(t, dir, 42)
```

Comment:
"It says, 'what do you get if you multiply six by nine?'"
"Six by nine. Forty two."
"That's it. That's all there is."
"I always thought something was fundamentally wrong with the universe."